homeassistant/components/onvif/camera.py

```diff
@@ -282,7 +282,7 @@ def setup_ptz(self):
         """Set up PTZ if available."""
         _LOGGER.debug("Setting up the ONVIF PTZ service")
         if self._camera.get_service("ptz", create=False) is None:
-            _LOGGER.warning("PTZ is not available on this camera")
+            _LOGGER.info("PTZ is not available on this camera")
```

Comment:
Let's make it debug. Also, "this" is useless in case you have 2 cameras.

Reply:
Done